cookstyle.gemspec

```diff
@@ -11,6 +11,7 @@ Gem::Specification.new do |spec|
   spec.email         = ['thom@chef.io', 'tsmith@chef.io']
   spec.summary       = 'Cookstyle is a code linting tool that helps you to write better Chef Infra cookbooks by detecting and automatically correcting style, syntax, and logic mistakes in your code.'
   spec.license       = 'Apache-2.0'
+  spec.homepage      = 'https://docs.chef.io/workstation/cookstyle/'
```

Comment:
Fixes this warning:

```
[2020-08-05 23:34:39 +0000] action built_in:build_gem started
WARNING:  no homepage specified
WARNING:  See http://guides.rubygems.org/specification-reference/ for help
  Successfully built RubyGem
  Name: cookstyle
  Version: 6.15.1
  File: cookstyle-6.15.1.gem
```

I figured if someone wants the gitbub repo that's easy enough to find.